Makefile

```diff
@@ -209,12 +205,10 @@ test-podspec:
 pod-lint:
 	(pod lib lint --allow-warnings --verbose --swift-version=5.1)
 
+# make check-version VERSION=3.1.0
+check-version:
+	(sh ./Script/version.sh $(VERSION))
 
-# make bump-versions from='3\.1\.0' to=3.1.1
-bump-versions:
-	(LC_ALL=C find . -type f -name 'project.pbxproj' -exec sed -i '' 's/$(from)/$(to)/' {} +)
-	(LC_ALL=C find . -type f -name '*.swift' -exec sed -i '' 's/$(from)/$(to)/' {} +)	
-	(LC_ALL=C find . -type f -name '*.podspec' -exec sed -i '' 's/$(from)/$(to)/' {} +)
-
+# make update-versions NAME=Core VERSION=3.8.0
 update-versions:
-	(sh ./Script/update-versions.sh -n Signal -v 3.7.4)
\ No newline at end of file
+	(sh ./Script/update-versions.sh -n $(NAME) -v $(VERSION))
```

Comment:
this make command was just used to test the update-versions script with specific values.  probably should be removed.